src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+let employeesList = [...document.querySelectorAll('li')];
+
+const helper = (list) => {
+  return list.map((x) => {
+    return {
+      salary: Number(x.dataset.salary.split(',').join('').slice(1)),
+      element: x,
+    };
+  });
+};
+
+employeesList = helper(employeesList);
+
+const sortList = (list) => {
+  return list.sort((a, b) => b.salary - a.salary);
+};
+
+employeesList = sortList(employeesList);
+
+const parentt = document.querySelector('ul');
```

Comment:
There is a typo in the variable name `parentt`. It should be `parent` to match the usage in line 24.